lsp-mode.el

```diff
@@ -809,7 +810,7 @@ Changes take effect only when a new session is started."
                                         (".*\\.tsx$" . "typescriptreact")
                                         (".*\\.ts$" . "typescript")
                                         (".*\\.jsx$" . "javascriptreact")
-                                        (".*\\.xml$" . "xml")
+                                        (".*\\.xml$" . "galaxy")
```

Comment:
This change is not fine - it will break the xml language servers. Are you sure it should go like this?

Reply:
whoops sorry, I committed this part without thinking. The language server does however process exclusively ".XML" files, which will definitely conflict with the default behaviour.

Is there a way to get around this? Or is this something that a galaxyproject user should override in their config?

Reply:
Either this or we can provide a hook in the language server so it can change the mappings in a buffer-local variable.